lib/challenge.js

```diff
@@ -0,0 +1,20 @@
+const { collection } = require("./database");
+
+async function getAllTasks() {
+  const results = await collection("challenge").find({}).toArray();
+  return results;
+}
+
+async function countAllTasks() {
+  const amount = await collection("challenge").count();
+  return amount;
+}
```

Comment:
you created a fucntion, doing exactly what you need for ..client/src/api/challenge.js  line 7, why not using it in a line?

Reply:
Good hint! THX